portfolio/src/main/java/com/google/sps/data/UserInfo.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data;
+
+/** Class that holds user's login information */
+public class UserInfo {
```

Comment:
Just checking, this class is only used for returning a JSON response back when the user makes a GET request to `/login` right?  If so I wonder if the code would be more easily understood if this was called something else, maybe LoginResponse or LoginInfo or UserLoginInfo?

I found myself getting a bit confused because there is both this class, as well as the "UserInfo" entity type, but I think they are different objects right, even though they have a very similar name?

Reply:
Yes, that makes sense - I originally created this class with the intent of holding the user's information like email. Looking back after my most recent edits, I _think_ I might not need this class anymore, as the "/login" servlet isn't called anywhere that I could find.

If so, this would clean things up nicely because I'm using the user's login Id to store the comments.

Reply:
`/login` is called in `script.js`, so this is still needed. I removed the email field and renamed the class to LoginInfo.